src/frameworks/index.ts

```diff
@@ -560,11 +567,12 @@ ${firebaseDefaults ? `__FIREBASE_DEFAULTS__=${JSON.stringify(firebaseDefaults)}\
       if (bootstrapScript) await writeFile(join(functionsDist, "bootstrap.js"), bootstrapScript);
 
       // TODO move to templates
+      const regionArg = ssrRegion !== DEFAULT_REGION ? `{region: "${ssrRegion}"}, ` : "";
```

Comment:
We should also add all the frameworksBackend stuff, e.g, `JSON.stringify(frameworksBackend)`